src/components/questions/search/Search.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+var Search = () => {
+  return (
+    <from>
```

Comment:
Is this a typo? Should be form not from.

Reply:
🤣 thanks!